client/transaction-pool/graph/src/ready.rs

```diff
@@ -44,6 +44,15 @@ pub struct TransactionRef<Hash, Ex> {
 	pub insertion_id: u64,
 }
 
+impl<Hash, Ex> parity_util_mem::MallocSizeOf for TransactionRef<Hash, Ex>
```

Comment:
Derive doesn't work due to extra bounds on `Hash` and `Ex`, right?

Reply:
It does work, actually